### DISCUSSION TITLE
Are Firefox beta versions automatically updated?

### DISCUSSION THREAD

**Discussion by gc435** [aclap-dev/video-downloadhelper#230] [Help and Q&A]
### Your question

I installed a beta of VDH on Firefox a few weeks ago.  I just checked, and I now seem to be running 8.1.1.3a1, which I think was released **_after_** I first installed the beta.

Does this mean that Firefox automatically updates when new beta versions are released?

### Addon & CoApp version and details

version: 8.2.1.3a1
channel: beta
build date: 2024-02-22
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): [redacted]
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "[redacted]",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by paulrouget** [comment]
> Does this mean that Firefox automatically updates when new beta versions are released?

Yes.

**Comment by GrampaWildWilly** [comment]
As Paul said, the betas will automatically update.  This happens when you have the browser settings for the VDH beta set to automatic update.  But be warned.  When you have automatic update enabled, Firefox can update the VDH beta WHILE YOU ARE DOWNLOADING SOMETHING.  Some users have reported this happening to them.  When this automatic update occurs, it interrupts any download you might have in progress, leaving you with a damaged file.  You'll have to redo such interrupted downloads again from the start.  For this reason, I have been recommending that you TURN OFF automatic updates for VDH in the browser settings.  The browser settings.  It's the browser that does automatic updates so you have to tell the browser not to do them.  When you turn off automatic updates for VDH, the browser settings will show you a button that allows you to check for updates.  When the check for updates says a new version is available, you then have the option of installing the new beta or not.  Paul has said he's working on a fix for this problem but until he releases it, I recommend you turn off automatic updates for VDH.  You'll have to remember to check for updates yourself, manually, every so often.  It's a tradeoff.
